Core/App/AppAPI.php

```diff
@@ -16,12 +16,20 @@
  * You should have received a copy of the GNU Lesser General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+
 namespace FacturaScripts\Core\App;
 
-use FacturaScripts\Core\Base\DataBase\DataBaseWhere;
 use FacturaScripts\Core\Model\ApiKey;
 use Symfony\Component\HttpFoundation\Response;
 
+\define('API_FOLDER', 'Dinamic\\Lib\\API\\');
```

Comment:
Define is used to create constants that are used throughout the application. This is not the case. Please revert these modifications.

Reply:
Reversed modifications.